src/routes/welcome/container/index.tsx

```diff
@@ -1,16 +1,12 @@
 import * as React from 'react'
```

Comment:
can we change this to `import React from 'react'`?

also, we can `import React, { ReactElement } from 'react'` so we move towards the react v17 (which won't need `React` to be imported anymore)